spec/libs/activity_runtime_api_spec.rb

```diff
@@ -328,6 +336,7 @@
             :drawing_prompt => '',
             :prompt => "the original prompt",  # this will be replaced.
             :is_required => false,             # this will be replaced.
+            :is_featured => false,             # this will be replaced.
```

Comment:
It took Dave and I a bit to figure out what 'this will be replaced' meant.  Changing to 'this will be replaced as the test runs' would make it more clear.